src/main/resources/grammar.hgr

```diff
@@ -41,13 +39,13 @@ grammar {
 
     # TODO: these should probably be partials...
```

Comment:
Yeah, I haven't quite finished "enums as partials" feature in Hermes yet, but when I find some time I'll implement that.  Continuous improvement!